CrazyCanvas/Include/Physics/PhysicsGroups.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Perhaps `CollisionGroups.h` is a more suitable name, to stay consistent with the name of the enum.

Reply:
Yup. my mind must have been somewhere else